bundle-workflow/tests/test_workflow/helper/test_utils.py

```diff
@@ -0,0 +1,13 @@
+import os
+
+
+class TestUtils:
+    def get_test_dependencies():
```

Comment:
Could you declare this function inside of the test class, if you foresee its reuse elsewhere please add comments or additional configuration

Reply:
Removed the common test utils and added it in the test class itself.